sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6389 @@
+                        "customPrivate": false,
+                        "json": "",
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                    },
+                    "unique": false,
+                    "errorLabel": "",
+                    "errors": "",
+                    "key": "cipCode",
+                    "tags": [],
+                    "properties": {},
+                    "conditional": {
+                        "show": null,
+                        "when": null,
+                        "eq": "",
+                        "json": ""
+                    },
+                    "customConditional": "",
+                    "logic": [
+                        {
+                            "name": "Disable CIP Code Input",
+                            "trigger": {
+                                "type": "javascript",
+                                "javascript": "result = data.hasOfferings;"
+                            },
+                            "actions": [
+                                {
+                                    "name": "Disable Input",
+                                    "type": "property",
+                                    "property": {
+                                        "label": "Disabled",
+                                        "value": "disabled",
+                                        "type": "boolean"
+                                    },
+                                    "state": true
+                                }
+                            ]
+                        }
+                    ],
+                    "attributes": {},
+                    "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "type": "textfield",
+                    "input": true,
+                    "refreshOn": "",
+                    "dataGridLabel": false,
+                    "addons": [],
+                    "inputType": "text",
+                    "id": "eeodhl7",
+                    "defaultValue": ""
+                },
+                {
+                    "label": "CIP digit",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "customClass": "",
+                    "modalEdit": false,
+                    "defaultValue": "",
+                    "persistent": true,
+                    "protected": false,
+                    "dbIndex": false,
+                    "encrypted": false,
+                    "redrawOn": "",
+                    "customDefaultValue": "",
+                    "calculateValue": "value = data.cipCode.substring(0, 2)",
```

Comment:
This value is consumed in just one place and the logic seems to be simple enough that it is not worth having a dedicated field to store it. Can you please consider removing it?

Reply:
Removed.